app/views/alaveteli_pro/plans/show.html.erb

```diff
@@ -11,9 +11,9 @@
       <div class="settings-left-column">
       </div>
       <div class="settings-right-column">
+        <%= form_tag(subscriptions_path) do %>
```

Comment:
Oh, this can be dropped too as its already in https://github.com/mysociety/alaveteli/commit/97f811844d8dd8315c3527b018622cd8bd3e3ca4

Reply:
👍